### PR TITLE
Update Service Mappings

### PR DESCRIPTION
**Pull request by scbedd**
We source this csv for information during our releases. [This PR](https://github.com/Azure/azure-docs-sdk-java/pull/1035/files#r441238276) is an example of the resulting update to docs.ms. 

This PR updates our service mapping such that the above PR is possible without manual intervention. 

I know I may be missing a few and please call those out, but I want to close this PR (at least this version!) quickly. I'll address any callouts immediately after merging though :) 

@Azure/azure-sdk-eng

### CODE REVIEW

File: _data/allpackages/dotnet-packages.csv [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-"Azure.Security.KeyVault.Certificates","Azure.Security.KeyVault.Certificates","4.0.2","","Replaces: Microsoft.Azure.KeyVault"
-"Azure.Security.KeyVault.Keys","Azure.Security.KeyVault.Keys","4.0.3","","Replaces: Microsoft.Azure.KeyVault"
-"Azure.Security.KeyVault.Secrets","Azure.Security.KeyVault.Secrets","4.0.3","","Replaces: Microsoft.Azure.KeyVault"
-"Azure.Storage.Blobs","Azure.Storage.Blobs","12.4.4","","Replaces: Microsoft.Azure.Storage.Blob"
-"Azure.Storage.Blobs.Batch","Azure.Storage.Blobs.Batch","12.2.1","",""
-"Azure.Storage.Common","Azure.Storage.Common","12.4.3","","Replaces: Microsoft.Azure.Storage.Common"
-"Azure.Storage.Files.DataLake","Azure.Storage.Files.DataLake","12.2.2","",""
-"Azure.Storage.Files.Shares","Azure.Storage.Files.Shares","12.2.3","","Replaces: Microsoft.Azure.Storage.File"
-"Azure.Storage.Queues","Azure.Storage.Queues","12.3.2","","Replaces: Microsoft.Azure.Storage.Queue"
+"Core","Azure.Core","1.2.2","",""
```
Comment (weshaggard) [inline _data/allpackages/dotnet-packages.csv:3]: I know that @AlexGhiondea and team explicitly wanted these to be the name that shows in the package table. Perhaps we need another column which we don't display directly but you use for your mapping. qa

Comment (weshaggard) [reply _data/allpackages/dotnet-packages.csv:3]: See https://github.com/Azure/azure-sdk/pull/1133.

**Comment by weshaggard** [comment]
Another question I have is should this go into the allpackages csv files or the new csv files? The new csv files also include things like the RepoPath so if you ever need to map or find the source location perhaps putting this information into the new csv file instead.

File: _data/allpackages/dotnet-packages.csv (hunk 1 above)
Comment (scbedd) [reply _data/allpackages/dotnet-packages.csv:3]: I cannot for the life of me understand why having two columns of _exactly_ the same content is a good idea. The offered PR offers no context either.

**Comment by scbedd** [comment]
@weshaggard what is the difference between the "new" packages and the "allpackages".

**Pull request closed without merging**